src/pages/index.tsx

```diff
@@ -2,6 +2,7 @@ import { Suspense, lazy } from "react";
 import { Navigate, Route, Routes } from "react-router-dom";
 import { LogTypes } from "constants/enums";
 import routes from "constants/routes";
+import NotFound from "./404";
```

Comment:
We should lazy load this since the graphic is pretty big we don't want to download it unless we need to.